implementations/rust/ockam/ockam_command/src/kafka/consumer/create.rs

```diff
@@ -0,0 +1,89 @@
+use std::net::SocketAddr;
+
+use clap::{command, Args};
+use colorful::Colorful;
+use ockam::{Context, TcpTransport};
+use ockam_api::{
+    nodes::models::services::{StartKafkaConsumerRequest, StartServiceRequest},
+    port_range::PortRange,
+};
+use ockam_core::api::Request;
+use ockam_multiaddr::MultiAddr;
+
+use crate::{
+    fmt_info,
+    kafka::{
+        kafka_consumer_default_addr, kafka_default_consumer_port_range,
+        kafka_default_consumer_server, kafka_default_project_route, parse_bootstrap_server,
+    },
+    node::NodeOpts,
+    service::start::start_service_impl,
+    util::node_rpc,
+    CommandGlobalOpts,
+};
+
+/// Create a new Kafka Consumer
+#[derive(Clone, Debug, Args)]
+pub struct CreateCommand {
+    #[command(flatten)]
+    node_opts: NodeOpts,
+
+    /// The local address of the service
+    #[arg(long, default_value_t = kafka_consumer_default_addr())]
+    addr: String,
```

Comment:
I wonder what's the point of changing the service address, the user shouldn't run more than one kafka service anyway. Can we make it static?

Reply:
I'm not entirely sure of the use case, however I do know that all of our other services allow you to define the address as well.

Removing it would remove consistency with the rest of them.

I think having a default like we do and allowing the use to not have to worry about it should be fine?